lua/nvim-ts-autotag/config/plugin.lua

```diff
@@ -141,7 +141,7 @@ local Setup = {
 }
 
 --- Do general plugin setup
----@param opts nvim-ts-autotag.PluginSetup
+---@param opts nvim-ts-autotag.PluginSetup|nil
```

Comment:
Instead of `|nil` can you use `?`
```suggestion
---@param opts nvim-ts-autotag.PluginSetup?
```

Just more in line with how we're passing optional arguments elsewhere.

Reply:
Oh nice, didn't know luadoc had that :+1: